### PR TITLE
WRP-3072: Updated dependencies to fix vulnerabilities

### PR DESCRIPTION
**Pull request by andreitirlalge**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Updated dependencies to fix vulnerabilities

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
All dependencies now use the latest minor version.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRP-3072

### Comments
Enact-DCO-1.0-Signed-off-by: Andrei Tirla [andrei.tirla@lgepartner.com](mailto:andrei.tirla@lgepartner.com)

### CODE REVIEW

File: packages/webostv/template/package.json [hunk 1]
```diff
@@ -33,17 +33,17 @@
...
-    "@enact/ui": "^4.6.0",
-    "@enact/webos": "^4.6.0",
-    "ilib": "npm:ilib-webos@^14.15.1-webos1",
+    "@enact/core": "^4.6.2",
+    "@enact/i18n": "^4.6.2",
+    "@enact/sandstone": "^2.6.3",
+    "@enact/spotlight": "^4.6.2",
+    "@enact/ui": "^4.6.2",
+    "@enact/webos": "^4.6.2",
+    "ilib": "^14.17.0",
```
Comment (daniel-stoian-lgp) [inline packages/webostv/template/package.json]:
```suggestion
    "ilib": "npm:ilib-webos@^14.17.0-webos1",
```

Comment (daniel-stoian-lgp) [reply packages/webostv/template/package.json]: please retest with the correct dependency

File: packages/theme/template/package.json [hunk 2]
```diff
@@ -28,20 +28,20 @@
...
+    "@enact/spotlight": "^4.6.2",
+    "@enact/ui": "^4.6.2",
     "classnames": "^2.3.2",
     "prop-types": "^15.8.1",
-    "ramda": "^0.28.0",
+    "ramda": "^0.29.0",
     "react": "^18.2.0"
   },
   "peerDependencies": {
     "ilib": "^14.15.1 || ^14.15.1-webos1"
```
Comment (daniel-stoian-lgp) [inline packages/theme/template/package.json]: please update this dependency also
ilib versions: https://www.npmjs.com/package/ilib?activeTab=versions
ilib-webos versions: https://www.npmjs.com/package/ilib-webos?activeTab=versions

**Review by daniel-stoian-lgp — COMMENTED** [review]
please check my comments

**Review by daniel-stoian-lgp — APPROVED** [review]

**Review by hong6316 — APPROVED** [review]
👍

**Pull request merged**